sim/tools.py

```diff
@@ -0,0 +1,416 @@
+"""Example tools.
+"""
+from __future__ import annotations
+
+import argparse
+import sys
+
+from typing import Literal, TextIO
+
+from .front import (
+    maze as maze_type,
+    size as size_type,
+    Tool,
+)
+from .maze import (
+    ascii_charset,
+    Charset,
+    Maze,
+    utf8_charset,
+    Walls,
+)
+
+
+MazeFormat = Literal['maze', 'maz', 'num', 'csv']
+MAZE_FORMATS = frozenset({'maze', 'maz', 'num', 'csv'})
```

Comment:
Not sure where those two are used...

Reply:
In deleted code obviously... 🙄

`MazeFormat` is still referenced in some documentation that also survived deletion.